courses/management/commands/gen_fake_data.py

```diff
@@ -30,8 +32,12 @@ def add_arguments(self, parser):
 
     def handle(self, *args, **options):
         record_count = 0
+        try:
+            bi = BackingInstance.objects.get(instance_url='https://edx.org')
```

Comment:
We use the factory in the exists branch, which sets up appropriate timestamps, urls, etc.  Not the end of the world to duplicate the config for this object.

Not sure if you guys missed that it's a `BackingInstanceFactory` instead of a `BackingInstance`. If you still want me to change, I'm happy to.